easybuild/easyconfigs/s/SIP/SIP-4.19.25-foss-2022a-Python-3.10.4.eb

```
@@ -0,0 +1,30 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Bart Verleye
+# Center for eResearch, Auckland
+easyblock = 'ConfigureMakePythonPackage'
+
+name = 'SIP'
+version = '4.19.25'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.riverbankcomputing.com/software/sip/'
+description = """SIP is a tool that makes it very easy to create Python bindings for C and C++ libraries."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+source_urls = ['http://sourceforge.net/projects/pyqt/files/sip/sip-%(version)s']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['b39d93e937647807bac23579edbff25fe46d16213f708370072574ab1f1b4211']
+dependencies = [('Python', '3.10.4')]
+
+configopts = "configure.py --bindir %(installdir)s/bin --incdir %(installdir)s/include "
+configopts += "--destdir %(installdir)s/lib/python%(pyshortver)s/site-packages"
+
+sanity_check_paths = {
+    'files': ['bin/sip', 'include/sip.h'] +
+             ['lib/python%%(pyshortver)s/site-packages/%s' % x
+              for x in ['sip.%s' % SHLIB_EXT, 'sipconfig.py', 'sipdistutils.py']],
+    'dirs': [],
+}
+
```

Something else that we recently started doing for new contributions, is adding a `sanity_check_commands` item here that can contain very simple checks for installed binaries, e.g. `app --help` or `app --version`.  This ensures that there are no basic issues with the binaries, e.g. missing libraries.

For SIP, it seems to be possible to do this as well:
```suggestion

sanity_check_commands = ['sip -h']

```